network-information/index.html

```diff
@@ -1,6 +1,6 @@
 ---
 feature_name: Network Information API
-chrome_version: 62
+chrome_version: 65
```

Comment:
@jeffposnick what's our precedent for updating the version in a demo? If a feature is amended later...?

Reply:
I think updating it with the correct minimum version to use the entirety of the sample is the precedent. That's what happened in https://github.com/tarunban/samples/commit/d3254545c43f86d5d1b102bd857836abaf066a26#diff-a318373df9ff649fbbc1264ae0e29971